backend/tests/github/api/organization_tests.py

```diff
@@ -0,0 +1,45 @@
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from apps.github.api.organization import OrganizationSerializer
+
+
+class TestOrganizationSerializer:
+    @pytest.mark.parametrize(
+        "organization_data",
+        [
+            {
+                "name": "GitHub",
+                "login": "github",
+                "company": "GitHub, Inc.",
+                "location": "San Francisco, CA",
+                "created_at": "2024-12-30T00:00:00Z",
+                "updated_at": "2024-12-30T00:00:00Z",
+            },
+            {
+                "name": "Microsoft",
+                "login": "microsoft",
+                "company": "Microsoft Corporation",
+                "location": "Redmond, WA",
+                "created_at": "2024-12-29T00:00:00Z",
+                "updated_at": "2024-12-30T00:00:00Z",
+            },
+        ],
+    )
+    @patch("apps.github.models.organization.Organization.objects.filter")
```

Comment:
Why do we need this patch here?
Could you add a comment explaining that?

Reply:
If I don't patch, the database will be queried.
![image](https://github.com/user-attachments/assets/736559a2-5508-4bea-9b24-7bbfebf41eb2)